openage/__main__.py

```diff
@@ -103,10 +102,17 @@ def main(argv=None):
     if args.no_devmode and args.devmode:
         cli.error("can't force enable and disable devmode at the same time")
 
-    if args.no_devmode:
-        config.DEVMODE = False
-    if args.devmode:
-        config.DEVMODE = True
+    try:
+        from . import config
+
+        if args.no_devmode:
+            config.DEVMODE = False
+        if args.devmode:
+            config.DEVMODE = True
+    except ImportError:
+        if args.no_devmode or args.devmode:
+            print("code was not yet generated, "
```

Comment:
Mainly because my editor window is not that wide, it ends at ~75 somewhere, so I try to break lines visually. It's 84 now, I changed it.